tests/wpt/web-platform-tests/html/syntax/serializing-html-fragments/serializing.html

```diff
@@ -58,6 +59,7 @@
 ["\"", "<span>\"</span>"],
 ["<style><&></style>", "<span><style><&></style></span>"],
 ["<script type=\"test\"><&><\/script>", "<span><script type=\"test\"><&><\/script></span>"],
+["<script type=\"test\"><&><\/script>", "<script type=\"test\"><&><\/script>"],
```

Comment:
This test performs `element.innerHTML` on each test element, so the `<script>` and `</script>` output should not be in the expected output.